roles/idrac_gather_facts/README.md

```diff
@@ -298,6 +667,7 @@ ansible.utils
       - PCIeSSDBackPlane
       - PowerSupply
       - PresenceAndStatusSensor
+      - SecureBoot
```

Comment:
can we update the examples to include displaying the output? (as in bspec)
Eg:
ansible.builtin.import_role:
    name: idrac_gather_facts
vars:
    << creds >>
    target:
      - System
      - Chassis
      - Manager
  tags:
    - idrac_gather_facts_out
 
-name: Print the System details
  ansible.builtin.debug:
    var: system
  tags:
    -idrac_gather_facts_out